frodo.js

```diff
@@ -1,128 +1,132 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!